Ago-Dic-2020/salazar-coronado-juan-daniel/practica2/practica2.py

```diff
@@ -0,0 +1,71 @@
+class Estudiante():
+    def __init__(self, nombre, edad, semestre, carrera):
+        self.nombre = nombre
+        self.edad = edad
+        self.semestre = semestre
+        self.carrera = carrera
+
+    def presentacion(self):
+        return "Soy " + self.nombre + ", tengo " + self.edad + " años y estoy en " + self.semestre + " semestre."
+
+    def getCarrera(self):
```

Comment:
El nombre de las funciones en python tiene que ser estilo `snake_case`, de acuerdo al estándar oficial: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Esto aplica para todas las funciones y variables que puedan ser actualizadas en esta práctica (y en futuras prácticas) 😄